resident/resident-service/src/main/java/io/mosip/resident/service/impl/DownloadCardServiceImpl.java

```diff
@@ -515,14 +521,18 @@ private ResidentTransactionEntity insertDataForVidCard(String vid, String uin)
 		return residentTransactionEntity;
 	}
 
-	private Map<String, Object> getVidDetails(String vid, String uin, int timeZoneOffset, String locale)
+	private Map<String, Object> getVidDetails(String vid, IdentityDTO identityDTOForDownloadableCardVid, int timeZoneOffset, String locale)
```

Comment:
In this method rename identityDTOForDownloadableCardVid to identityDTO

Reply:
done.